File: Tugas5/TugasFPC/tugas5.1bilanganganjilgenap.pas
{Nama           :Hilda Nurhidayati}
{NPM            :20.14.1.0039}
{Nama Program   :tugas5.1ganjilgenap}

program bilangan_ganjil_genap;
uses crt;

function bilangan (x, y : integer) : integer;
begin
if x mod y = 0 then
writeln('Bilangan ini adalah bilangan genap')
else
writeln('Bilangan ini adalah bilangan ganjil');
end;

var
a : integer;

begin
clrscr;
write('Masukkan Bilangan : ');readln;
write(bilangan(a,4));
readln;

end.

File: Tugas5/C++/Tugas5.1bilanganganjilgenap.cpp
//Nama			:Hilda Nurhidayati
//NPM			:20.14.1.0039
//Nama Program  :Bilangan Ganjil Genap 

#include<iostream>
using namespace std;

int n;
int hasil (int a){
	int hasil;
	hasil = n%4 == 0;
}
int main ()
{
	cout<<"Program Bilangan Ganjil Genap"<<endl;
	cout<<"Masukkan Nilai : ";
	cin>>n;
	cout<<endl;
	
	if (hasil(n)){
		cout<<"Bilangan ini adalah bilangan genap";
	}else {
		cout<<"Bilangan ini adalah bilangan ganjil";
	}
	return 0;
}
